opm/simulators/wells/WellStateFullyImplicitBlackoil.hpp

```diff
@@ -264,12 +264,6 @@ namespace Opm
                             wellReservoirRates()[ idx ] = prevState->wellReservoirRates()[ oldidx ];
                         }
 
-                        // Well potentials
-                        for( int i=0, idx=newIndex*np, oldidx=oldIndex*np; i<np; ++i, ++idx, ++oldidx )
-                        {
-                            wellPotentials()[ idx ] = prevState->wellPotentials()[ oldidx ];
-                        }
-
```

Comment:
Is this a mismerge or did you intend to remove this block of code?

Reply:
Yes it is  mismerge. I will update asap.